imports/client/components/HuntApp.tsx

```diff
@@ -139,7 +138,7 @@ const HuntApp = React.memo((props: RouteComponentProps<HuntAppParams>) => {
       _id: huntId,
     });
     const user = Meteor.user();
-    const member = !!user && _.contains(user.hunts, huntId);
+    const member = !!user && user.hunts.includes(huntId);
```

Comment:
Careful -- I think we also need to guard `user.hunts` being undefined.

Reply:
Is that true? It's not defined as an optional property

Reply:
Its seems obvious to me that anything that we're attaching to `Meteor.user()` is going to have *some* transient state where the property isn't present.

I agree that most users will have hunts with at least one hunt, as a result of other flows calling `Meteor.users.update`, but the first user obviously won't until they create a hunt.  So yeah, I think the type signature is overly optimistic.

Reply:
Fixed by changing the definition to be optional and fixing the fallout